cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
```

Comment:
- Isn't it better to put _**payment**_ object to ViewModel?
- Additionally, even if use here then you cannot modify **_let_** constants and you need to initialise it if it is optional. 
```suggestion
    var payment: Payment?
```